static/js/modules/download.js

```diff
@@ -137,7 +137,7 @@ DownloadItem.prototype.refresh = function() {
     contentType: 'application/json',
     dataType: 'json'
   });
-  this.promise.then(this.handleSuccess.bind(this));
+  this.promise.done(this.handleSuccess.bind(this));
```

Comment:
Yep, I was confused about this too. But there was an issue with the wrong context not being passed and this solution fixed it. Since it's a test and not a feature, seemed ok to just leave it.